CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to secrets-store-csi-driver-provider-gcp will be documented
 
 ## unreleased (v0.4.0)
 
+### Added
+
+* `-write_secrets` flag. Set `-write_secrets=false` with `v0.0.21+` of the `secrets-store-csi-driver` to have the driver write the secrets. This will be made the default in `v0.5.0`. [#98](https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp/issues/98)
```

Comment:
`...driver write the secrets.` I thought that 'false' meant that the driver would return the Secrets in the grpc response?

Reply:
The "driver" is the `secrets-store-csi-driver` process who made the RPC to our process. So when we return it in the grpc response the driver, not `secrets-store-csi-driver-provider-gcp`, writes the files. Updated to try and clarify